crates/core/src/view/keyboard.rs

```diff
@@ -25,6 +29,47 @@ pub struct Layout {
     pub widths: Vec<Vec<f32>>,
 }
 
+impl Default for Layout {
```

Comment:
Added a default Keyboard Layout to make unit tests simpler, since rust does not have robust mocking